docs/en/guides/How-to-build.md

```diff
@@ -65,6 +65,8 @@ a oap image with name `bar/oap:foo`, run the following command
 > HUB=bar TAG=foo make docker.oap
 
 ## Setup your IntelliJ IDEA
+**NOTICE**: If you clone the codes from Github, please make sure that you had finished step 1 to 7 in section **[Build from GitHub](#build-from-apache-source-code-release)**.
```

Comment:
Also, text is not build from GitHub, you are buildin from source codes.

Reply:
yes. i will fix later